### PR TITLE
ofr-12 Created login page

### PR DESCRIPTION
**Pull request by QriusDev**

### CODE REVIEW

File: outfitter-mobile-app/app.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "expo": {
+    "name": "outfitter-mobile-app",
+    "description": "This project is really great.",
```
Comment (kylemart) [inline outfitter-mobile-app/app.json]: Can we change the description here?

Comment (kylemart) [inline outfitter-mobile-app/app.json]: Can we change the description here?

File: outfitter-mobile-app/App.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { StyleSheet, Text, View, TextInput, Button, ToastAndroid, TouchableHighlight } from 'react-native';
+import { Header } from 'react-native-elements';
+
+export default class App extends React.Component {
+  
+  constructor(props){
```
Comment (kylemart) [inline outfitter-mobile-app/App.js]: Add space between ')' and '{'

File: outfitter-mobile-app/App.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    super(props);
+    this.login = this.login.bind(this);
+    this.state = {user: '', pass: ''};
+  }
+
+  login(){
```
Comment (kylemart) [inline outfitter-mobile-app/App.js]: Add space between ')' and '{'

File: outfitter-mobile-app/.gitignore [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+local.properties
+
+# node.js
+node_modules/
+npm-debug.log
+package-lock.json
+
+# BUCK
+buck-out/
+\.buckd/
```
Comment (kylemart) [inline outfitter-mobile-app/.gitignore:46]: Was this automatically added? If not, what is the significance of the leading '\'?

File: outfitter-mobile-app/App.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+            placeholder="password" 
+            textContentType="password"
+            secureTextEntry={true}
+            placeholderTextColor="#90a4ae"
+            onChangeText={(pass) => {this.setState({pass})}}
+            />
+          <TouchableHighlight
+            style={styles.loginButton}
+            onPress={this.login}
+            >
```
Comment (kylemart) [inline outfitter-mobile-app/App.js:47]: Missing '/'

**Review by kylemart — CHANGES_REQUESTED** [review]
Everything looks good, but please address my comments when you get the chance.

File: outfitter-mobile-app/App.js [hunk 6]
```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { StyleSheet, Text, View, TextInput, Button, ToastAndroid, TouchableHighlight } from 'react-native';
+import { Header } from 'react-native-elements';
+
```
Comment (Brantech) [inline outfitter-mobile-app/App.js:8]: Even though it is obvious, add a comment stating that this is the main container for the ui

**Review by Brantech — CHANGES_REQUESTED** [review]
See my comment. Other than that, good to go.

**Comment by kylemart** [comment]
Can we get an update on the above change requests, @GearTech0 ? 

Also, pinging Agent @sclong: don't forget to submit your review for this (when you get the chance).

**Review by sclong — APPROVED** [review]

File: outfitter-mobile-app/.gitignore (hunk 4 above)
Comment (QriusDev) [reply outfitter-mobile-app/.gitignore:46]: Auto added

File: outfitter-mobile-app/App.js (hunk 3 above)
Comment (QriusDev) [reply outfitter-mobile-app/App.js]: coolio

File: outfitter-mobile-app/App.js (hunk 5 above)
Comment (QriusDev) [reply outfitter-mobile-app/App.js:47]: that doesnt need one. there is a nested text element inside of it. ???

Comment (kylemart) [reply outfitter-mobile-app/App.js:47]: Missed the closing tag when looking through. You're correct.

File: outfitter-mobile-app/app.json (hunk 1 above)
Comment (kylemart) [reply outfitter-mobile-app/app.json]: Commit 9caac9d0de137eeddd510d363c661e73dbc931ed hasn't addressed this. Are we planning on keeping this description as-is? @GearTech0

File: outfitter-mobile-app/.expo/packager-info.json [hunk 7]
```diff
@@ -1,8 +1,8 @@
 {
   "devToolsPort": 19002,
-  "expoServerPort": 19000,
-  "packagerPort": 19001,
-  "packagerPid": 10749,
+  "expoServerPort": null,
```
Comment (kylemart) [inline outfitter-mobile-app/.expo/packager-info.json:3]: Should these be null? @GearTech0

File: outfitter-mobile-app/app.json (hunk 1 above)
Comment (QriusDev) [reply outfitter-mobile-app/app.json]: oh yeah forgot about this. What description do we want?

File: outfitter-mobile-app/.expo/packager-info.json (hunk 7 above)
Comment (QriusDev) [reply outfitter-mobile-app/.expo/packager-info.json:3]: theyre auto gen. i assume so...

File: outfitter-mobile-app/app.json (hunk 1 above)
Comment (kylemart) [reply outfitter-mobile-app/app.json]: Would it hurt just to delete it? Lol

**Review by kylemart — APPROVED** [review]

**Review by Brantech — APPROVED** [review]

**Pull request closed without merging**